New package: SciMLJacobianOperators v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: SciMLJacobianOperators
- Repository: https://github.com/SciML/NonlinearSolve.jl
- Created by: @avik-pal
- Version: v0.1.0
- Commit: 05aa3db8351e2da44729e99795bd2b30f7c7a601
- Reviewed by: @avik-pal
- Reference: https://github.com/SciML/NonlinearSolve.jl/commit/05aa3db8351e2da44729e99795bd2b30f7c7a601#commitcomment-147209784
- Description: High-performance and differentiation-enabled nonlinear solvers (Newton methods), bracketed rootfinding (bisection, Falsi), with sparsity and Newton-Krylov support.
<!-- 482abc00e933fd63777b9654337e0e44b158ece2b0f4a87375d6b37e18ee89d213c289ea4020f02f4bad9581687305af8e0af49bc38cd04243f37ba1fac13c0eb4e0ef5ddc42453781a0aa3b3aacaf3175136cd40bfba2e941132f3dfb5f3e95d8d74bbea69d0d644c4feb735ebd64b84df607c9dd6cec35e8e4926b8bd338ad75fc31ebf4718c5eb37f0b833f450157bd956df4de391edb99d6903988f2144978caa8569c0a0a7de7ea82401908d761cb70deb92befa9a0be2c82e92442a10ccf6fa17d9cbf97277344268e59615baca2f149d04c4ee6ce114b92bb86829e42f3f651d2b53deb88afefcf3e5d6b6e19c8f34b682701958c7ad494d2b1a46b48 -->

goerz commented:
Could you add a minimal README to the `lib/SciMLJacobianOperators` subfolder that explains the purpose/scope of the package and the relation to the main `NonlinearSolve` package, in a sentence or two?

avik-pal commented:
[noblock] Added a README for the subpackage. There are docs as well but waiting for the sciml multidocs to pick it up

JuliaRegistrator committed:
New package: SciMLJacobianOperators v0.1.0

UUID: 19f34311-ddf3-4b8b-af20-060888a46c0e
Repo: https://github.com/SciML/NonlinearSolve.jl.git
Tree: 991d2a8900e687e2c693d587daa739c8fda01366

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
